hardware/VirtualThermostat.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include <map>
```

Comment:
map should not need to be included here, it is already included in stdafx.h

Reply:
ok done